ddl/ddl.go

```diff
@@ -388,7 +388,7 @@ func (d *ddl) CreateTable(ctx context.Context, ident table.Ident, colDefs []*col
 	is := d.GetInformationSchema()
 	schema, ok := is.SchemaByName(ident.Schema)
 	if !ok {
-		return errors.Trace(qerror.ErrDatabaseNotExist)
+		return terror.Schema.New(terror.DatabaseNotExists, "database %s not exists", ident.Schema)
```

Comment:
I considered to add `Cause` error as a field in Error struct, but haven't figure out the API.